lib/getNextJobs.js

```diff
@@ -21,6 +21,9 @@ const getNextJobs = (workflowGraph, config) => {
         throw new Error('Must provide a PR number with "~pr" trigger');
     }
 
+    // Check if the job is triggerd by prChain with regexp
+    const prChainTrigger = config.trigger.match(/^(PR-[0-9]+:)?[\w-]+$/u);
```

Comment:
I think we should use a regexp of dataschema instead of defined here if we can.

Reply:
Ok, I will use data-schema.

Reply:
I fixed to use data-schema